web/src/games/kriegspiel/instructions.md

```diff
@@ -0,0 +1,55 @@
+# Le Jeu de la Guerre
```

Comment:
This game doesnt look like it is going to play well in mobile phones. If you don't plan on supporting mobile phones, please highlight it here on the instructions.

Reply:
I have mobile phone UI control support, by using the react-use-gesture, should I load this library?

Reply:
Ok, lets follow up on this later after we upgrade react.